validator/src/cli.rs

```diff
@@ -297,6 +332,17 @@ fn deprecated_arguments() -> Vec<DeprecatedArg> {
             .help("Skip ledger verification at validator bootup."),
         replaced_by: "skip-startup-ledger-verification",
     );
+    // Deprecated as of v2.2
+    add_arg!(
+        Arg::with_name("tower_storage")
+            .long("tower-storage")
+            .possible_values(&["file", "etcd"])
+            .default_value("file")
```

Comment:
Similar to other comment, value copied over to avoid plumbing `DefaultArgs`:
https://github.com/anza-xyz/agave/blob/e9e0e40958c87275613a32c26cc52db9d6608fd6/validator/src/cli.rs#L434